Audit mode (#1805)
linkerd/website: linkerd.io/content/2.16/features/server-policy.md

Note that dynamically changing the policy to deny existing connections may
result in an abrupt termination of those connections.

## Audit mode

> A [`Server`]'s default policy is declared in its `accessPolicy` field, which

Reviewer: ```suggestion
A [`Server`]'s default policy is defined by its `accessPolicy` field, which
```

??

defaults to `deny`. That means that, by default, traffic that doesn't conform to
the rules associated to that Server is denied (the same applies to `Servers` that

> don't have associated rules yet). This can inadvertently sever traffic if you

Reviewer: ```suggestion
don't have associated rules yet). This can inadvertently prevent traffic if you
```
